org/views.py

```diff
@@ -97,3 +97,50 @@ def AddVolunteer(request,org_id):
             return Response({"message":"You are added as a volunteer"},status.HTTP_201_CREATED)
     else:
         return Response({"detail":"Organization not present"},status.HTTP_400_BAD_REQUEST)
+
+
+@swagger_auto_schema(
+    operation_id="view_list_of_permissions",
+    method='GET',
+    responses={
+        '200': set_example(responses.list_permission_set_200),
+        '401': set_example(responses.user_unauthorized_401),
+        '404': set_example({"detail": "This organisation doesn't exist."}),
+        '400': set_example({"detail" : "You are not a member of this organisation"}),
+        '403': set_example({"detail": "You are not authorised to view this."}),
+    }
+)
+@api_view(['GET'])
+@permission_classes([IsAuthenticated])
+def Permission_Set_List(request,org_id):
+    try:
+        org = Org.objects.get(pk=org_id)
+    except Org.DoesNotExist:
+        return Response({"detail":"This organisation doesn't exist."}, status.HTTP_404_NOT_FOUND)
+    try:
+        member = Member.objects.get(
+        user = request.user,
+        org = org
+        )
+    except Member.DoesNotExist:
+        return Response({"detail" : "You are not a member of this organisation"}, status.HTTP_400_BAD_REQUEST)
+
+
+    if member.permissions.permissions.permissions[1]: 
```

Comment:
```suggestion
    from org.custom_model_field import PermissionSet # on the top 
    if member.permissions.permissions.permissions[PermissionSet.IS_STAFF]: 
```

will be better way of doing this. 

We will rename **this ridiculous thing `permissions.permissions.permissions` in a separate issue lol**

Reply:
Please create this as a separate **good first issue**.

Reply:
> will be better way of doing this.
>

I guess there is an ambiguity between the PermissionSet class and PermissionSet model due to which the tests fail when I make this change.. 😅 
`AttributeError: type object 'PermissionSet' has no attribute 'objects'` pops up when I refer to PermissionSet model below to get the respective ids and names.

Reply:
> Please create this as a separate **good first issue**.
>
I think it is #92 what you're referring to..? 😅

Reply:
> > will be better way of doing this.
> 
> I guess there is an ambiguity between the PermissionSet class and PermissionSet model due to which the tests fail when I make this change.. 😅
> `AttributeError: type object 'PermissionSet' has no attribute 'objects'` pops up when I refer to PermissionSet model below to get the respective ids and names.

Once this PR is merged, I'll change it accordingly when I change `custom_model_field.PermissionSet` to `custom_model_field.Permissions` while working on #92 , Does this sound right..?

Reply:
> I guess there is an ambiguity between the PermissionSet class and PermissionSet model due to which the tests fail when I make this change.. sweat_smile
> AttributeError: type object 'PermissionSet' has no attribute 'objects' pops up when I refer to PermissionSet model below to get the respective ids and names.

Yup ambiguity. No need to create new issues. You can go on with #92. And u can resolve this conversation once u see it.